terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/gh-runners.ts

```diff
@@ -377,26 +377,26 @@ export async function getRunnerTypes(
           return;
         }
 
-        if (runnerType.variants.size > 0) {
-          Array.from(runnerType.variants.keys()).forEach((variant) => {
-            const variantType = runnerType.variants?.get(variant);
-            /* istanbul ignore next */
-            if (!variantType) {
-              return;
-            }
-
-            let variantRunnTypeName: string;
-            if (key.startsWith('lf.c.')) {
-              variantRunnTypeName = `lf.c.${variant}.${key.slice(5)}`;
-            } else if (key.startsWith('lf.')) {
-              variantRunnTypeName = `lf.${variant}.${key.slice(3)}`;
-            } else {
-              variantRunnTypeName = `${variant}.${key}`;
-            }
-
-            result.set(variantRunnTypeName, { ...runnerType, ...variantType, runnerTypeName: variantRunnTypeName });
-          });
-        }
+        Array.from(runnerType.variants.keys()).forEach((variant) => {
+          const variantType = runnerType.variants?.get(variant);
+          /* istanbul ignore next */
+          if (!variantType) {
+            return;
+          }
+
+          let variantRunnTypeName: string;
+          if (key.startsWith('lf.c.')) {
```

Comment:
We should probably document what this behavior does somewhere, the logic looks alright but a plain english explanation might help someone decipher this quicker.

Reply:
I'll add a note on our GHA infra runbook. But we do have a ci that validates the scale-config and give useful error messages :)

Reply:
@ZainRizvi @seemethere 

updated the relevant section in our runbook: https://docs.google.com/document/d/1nq3dx-_8wasii1koCkXJDSo3uz_0Ee8DzIS2-j2TOpA/edit?tab=t.0#heading=h.qpdlbrimw1ak